example/nginx-letsencrypt.conf

```diff
@@ -0,0 +1,39 @@
+# Example configuration for Django Polaris using Nginx as reverse proxy with HTTPS enabled.
```

Comment:
Can you add a comment specifying the linux distribution used? The only reason this is necessary is due to the file-system paths generated by Lets Encrypt, which may not be the correct paths on other linux distributions.

Lets also mention that this is not used in the deployment of this example application, but is purely for reference.

Reply:
Done, thanks